Makefile

```diff
@@ -182,55 +178,48 @@ test-e2e-buildah-openshift: bats
 
 # runs the end-to-end tests for s2i-python
 .PHONY: test-e2e-s2i-python
-test-e2e-s2i-python: prepare-e2e-s2i
```

Comment:
Same question I have too. I mean these targets are used for running the e2e for each language of s2i separately right? Everytime we may not require to run e2e for all languages of s2i when we are upgrading a particular s2i task say when a change is specific to s2i-java only.

Reply:
prepare-e2e-s2i target was removed in one of the previous PRs, because we shifted from PVC to volumeClaimTemplate i believe and we no longer needed to apply that PVC, so there was no prepare step occuring, that was just redundant & gave misinformation here, so removed it.